hendrix/deploy/base.py

```diff
@@ -35,17 +35,21 @@ def __init__(self, action='start', options={}, reactor=reactor):
         # because running the management command overrides self.options['wsgi']
         if self.options['wsgi']:
             if hasattr(self.options['wsgi'], '__call__'):
-                # If it has a __call__, we assume that it is the application object itself.
+                # If it has a __call__, we assume that it is the application
+                # object itself.
                 self.application = self.options['wsgi']
                 try:
-                    self.options['wsgi_app_name'] = "%s.%s" % (self.application.__module__, self.application.__name__)
+                    self.options['wsgi'] = "%s.%s" % (
```

Comment:
The point of this change was to accommodate whatever django is doing in their management command magic whilst not adding a new option. This way the user interface doesn't have to change and the skip on the test can be removed. `options` was only ever supposed to have user inputs/defaults entered from the terminal - at least in my mind. Not sure why the Django peeps would put a callable in there.